proto/session_server.proto

```diff
@@ -19,6 +19,9 @@ message SessionParams {
 
   // Information about client tunnel endpoint.
   TunnelEndpoint endpoint = 4;
+
+  // Openflow port on which device was learned
```

Comment:
No need to mention that it's open-flow -- it's just the port number. And it's not "was learned" -- it's just where it *is* (the fact that it was done through learning is immaterial here).

Reply:
Fixed.